### DISCUSSION TITLE
Not able to setup

### DISCUSSION THREAD

**Discussion by ishanvaghani** [KRTirtho/spotube#998] [Q&A]
[SEVERE] envied_generator:envied on lib/collections/env.dart (cached):

Environment variable file doesn't exist at `.env`.
package:spotube/collections/env.dart:7:16
  ╷
7 │ abstract class Env {
  │                ^^^
  ╵
[SEVERE] Failed after 104ms

**Getting above error can someone help?**

**Comment by KRTirtho** [comment]
The build steps/contribution guidelines were explained in the README and CONTRIBUTION.md
https://github.com/KRTirtho/spotube/blob/master/CONTRIBUTION.md#your-first-code-contribution

**Reply by zhuyeaini9** [reply]
I have created the .env file and copy the example.env's content to it,but still have same error.

**Comment by ishanvaghani** [comment]
How can i get SPOTIFY_SECRETS?

**Reply by KRTirtho** [reply]
Follow spotify developer docs https://developer.spotify.com/documentation/web-api

**Comment by ishanvaghani** [comment]
SPOTIFY_SECRETS=clintId1:clientSecret1,clientId2:clientSecret2

i'm only getting one client id and one client secret. Here it says 2 how?

**Reply by KRTirtho** [reply]
If you've one pair, just put one pair. You can put multiple pairs separated by commas, that is what the comment meant.

**Reply by ishanvaghani** [reply]
Feed is not loading
